functions/colors.zsh

```diff
@@ -79,6 +79,10 @@ function getColorCode() {
     else
       echo -n "$1"
     fi
+  # Check if value is none with any case.
+  elif [[ $1 = [nN][oO][nN][eE] ]]
```

Comment:
It would be more readable to write that as `elif [[ "${(L)1}" == "none" ]]`

Reply:
Yes I add to the next commit